test/bdd/fixtures/auth-rest/docker-compose.yml

```diff
@@ -23,6 +23,7 @@ services:
       - AUTH_REST_GOOGLE_CLIENTSECRET=TODO
       - AUTH_REST_SDS_URL=TODO        # onboard user: https://github.com/trustbloc/hub-auth/issues/38
       - AUTH_REST_KEYSERVER_URL=TODO  # onboard user: https://github.com/trustbloc/hub-auth/issues/38
+      - AUTH_REST_HYDRA_URL=http://TODO.com      # setup Hydra in BDD tests: https://github.com/trustbloc/hub-auth/issues/52
```

Comment:
I would add the TODO word here to be consistent with the other TODOs
(and I guess that would apply to the TODOs above these two lines as well...)

Reply:
fixed